src/Xunit.SkippableFact.Tests/Xunit.SkippableFact.Tests.csproj

```diff
@@ -17,7 +17,4 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Please revert this change. VS will just put it back if we remove it.

Reply:
VS won't revert it :) They've fixed it in 15.7: https://github.com/Microsoft/vstest/issues/472